components/AgentForm.tsx

```diff
@@ -152,6 +157,9 @@ const Tool = (props: {
 };
 
 export default function BaseForm(props: Props) {
+  const router = useRouter();
+  const {agentId} = router.query;
```

Comment:
Pass agentId by props instead. This form should not depend on query parameters

Reply:
sure it has sth to do with a bad rebase